configs/templates/unified-dev/spack.yaml

```diff
@@ -28,7 +28,7 @@ spack:
       # Various fms tags (list all to avoid duplicate packages)
       - fms@release-jcsda
       - fms@2023.01
```

Comment:
Did we want to remove `fms@2023.01`?

Reply:
I was going to but then noticed it's hardcoded into ufs-srw-app and one other metapackage, so for now I'd prefer to not go down that rabbit hole unless necessary.